webapp/src/Controller/BaseController.php

```diff
@@ -394,6 +402,10 @@ protected function deleteEntities(
             $descriptions[] = $entity->getShortDescription();
         }
 
+        if (count($entities) > 1) {
+            $readableType = (substr($readableType, -1) !== 'y') ? $readableType.'s' : substr($readableType, 0, -1).'ies';
```

Comment:
Maybe swap the two parts and change the `!==` to `===`?
Also maybe add some braces like `($readableType.'s')` for readability?
Also spaces.